lib/perlcritic/Perl/Critic/Policy/HashKeyQuotes.pm

```diff
@@ -0,0 +1,35 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package Perl::Critic::Policy::HashKeyQuotes;
+
+use strict;
+use warnings;
+use base 'Perl::Critic::Policy';
+
+use Perl::Critic::Utils qw( :severities :classification :ppi );
+
+our $VERSION = '0.0.1';
+
+sub default_severity { return $SEVERITY_HIGH }
+sub default_themes { return qw(openqa) }
+sub applies_to { return qw(PPI::Token::Quote::Single PPI::Token::Quote::Double) }
+
+# check that hashes are not overly using quotes
+# (os-autoinst coding style)
+sub violates {
+    my ($self, $elem) = @_;
```

Comment:
I suggest to use signatures, e.g.
```suggestion
sub violates ($elem) {
```
in new code.

Reply:
Where's the `$self` though?

Reply:
Oh, my bad, didn't read :sweat: Fixing it now

Reply:
Strictly, this isn't a new file, though, just a move, and personally I wouldn't bother to add signatures to plugins, but ok...

Reply:
Sorry, I've forgot the `$self` in my suggestion.